package.json

```diff
@@ -32,6 +32,7 @@
     "immer": "^9.0.16",
     "lodash-es": "^4.17.21",
     "material-icons": "^1.13.1",
+    "moment": "^2.29.4",
```

Comment:
The moment dev team considers it a legacy project and they are discouraging its use in new projects. See https://momentjs.com/docs/

I've already created a `formatDate()` function in `utils.ts` that does "d MMM yyyy" format using native code. It shouldn't be too hard to do something similar for the time portion. Or you could use `date-fns`, which we are already including as a peer dependency for chartjs. It has a [format ](https://date-fns.org/v2.29.3/docs/format) function.